rapids-cmake/cpm/logger.cmake

```diff
@@ -0,0 +1,67 @@
+#=============================================================================
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_cpm_logger
+-----------------
+
+.. versionadded:: v25.02.00
+
+Allow projects to build `rapids-logger` via `CPM`.
+
+Uses the version of rapids-logger :ref:`specified in the version file <cpm_versions>` for consistency
+across all RAPIDS projects.
+
+Unlike most `rapids_cpm` functions, this one does not support export sets because rapids-logger adds targets directly to the calling project's own export set and does not require its own exporting or to be found at all by consuming projects once the first project's call to rapids_make_logger has completed.
+
+.. code-block:: cmake
+
+  rapids_cpm_logger( [<CPM_ARGS> ...])
+
+.. |PKG_NAME| replace:: logger
+.. include:: common_package_args.txt
+
+Result Functions
+^^^^^^^^^^^^^^^^
+  :cmake:command:`rapids_make_logger` is made available
+
+#]=======================================================================]
+function(rapids_cpm_logger)
```

Comment:
This should be `rapids_cpm_rapids_logger`, since the name of the package is "rapids-logger". This leads to confusing patterns elsewhere, because we just get the word "logger" (like in `logger.cmake`) without a clear indication that this is referring to a RAPIDS-maintained rapids-logger package.

Reply:
Done.